### PR TITLE
Clarify difference between filled_area, area, and convex_hull_area in regionprops

### PR DESCRIPTION
**Pull request by tseclaudia**
## Description

Addresses #1961

## References

Issue #1961

### CODE REVIEW

File: skimage/measure/_regionprops.py [hunk 1]
```diff
@@ -349,7 +349,6 @@ def __eq__(self, other):
 
 def regionprops(label_image, intensity_image=None, cache=True):
     """Measure properties of labeled image regions.
-
```
Comment (soupault) [inline skimage/measure/_regionprops.py]: Single blank lines before and between docstring sections are mandatory.
Please, revert this one and the similar changes below.

File: skimage/measure/_regionprops.py [hunk 2]
```diff
@@ -383,7 +379,9 @@ def regionprops(label_image, intensity_image=None, cache=True):
     **centroid** : array
         Centroid coordinate tuple ``(row, col)``.
     **convex_area** : int
-        Number of pixels of convex hull image.
+        Number of pixels of convex hull image, which is the smallest convex 
+        polygon that encloses the region. The size of the convex hull image is 
+        described by the bounding box.
```
Comment (soupault) [inline skimage/measure/_regionprops.py]: Is not clear if the attribute store the number of pixels of convex polygon or of bounding box.

**Comment by tseclaudia** [comment]
I've made the requested changes.

File: skimage/measure/_regionprops.py [hunk 3]
```diff
@@ -349,7 +349,7 @@ def __eq__(self, other):
 
 def regionprops(label_image, intensity_image=None, cache=True):
     """Measure properties of labeled image regions.
-
+
```
Comment (soupault) [inline skimage/measure/_regionprops.py]: In `skimage` we use a convention for blank lines not to contain any whitespaces.
Please, change the commit accordingly.

**Comment by soupault** [comment]
Closed in advance of #3602 .

**Pull request closed without merging**